packages/ui/components/Drawer.tsx

```diff
@@ -27,8 +27,16 @@ function Drawer({
   expanded,
   handleCloseClick,
   closeDrawer,
```

Comment:
We can set a default value here of `closeDrawer=()=>{}` so we don't need to pass in an empty function in our other components when we want to use `Drawer`